keyboards/gh60/readme.md

```diff
@@ -5,8 +5,8 @@ GH60
 
 A common 60% PCB.
 
-Keyboard Maintainer: QMK Community
-Hardware Supported: GH60 PCB
+Keyboard Maintainer: QMK Community  
+Hardware Supported: GH60 PCB  
```

Comment:
please specify which revisions of the GH60

Reply:
This will require more research on my part. So far I've seen nothing to indicate that the firmware is only valid for specific revisions.